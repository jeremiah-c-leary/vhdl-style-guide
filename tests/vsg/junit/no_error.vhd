
architecture rtl of fifo is

begin

end architecture rtl;
