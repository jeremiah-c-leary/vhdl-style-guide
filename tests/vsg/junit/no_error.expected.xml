<?xml version="1.0" ?>
<testsuite errors="0" hostname="Rimmer" failures="0" timestamp="2018-12-07T03:10:28" tests="1" time="0" name="vhdl-style-guide">
  <properties>
  </properties>
  <testcase name="tests/vsg/junit/no_error.vhd" time="0">
  </testcase>
  <system-out>
  </system-out>
  <system-err>
  </system-err>
</testsuite>
